src/lambda.rs

```diff
@@ -0,0 +1,94 @@
+mod commp;
+
+use std::error::Error;
+use std::str::FromStr;
+
+#[macro_use]
```

Comment:
In Rust editiion 2018, you can import macros directly using normal imports. So you don't need to use `#[macro_use]` or `extern crate` anymore. I guess the `lamda!` macro comes from this crate, so it would just be:

```rust
use lambda_runtime::lamda
```

in order to import the lamnda macro.